package.json

```diff
@@ -71,6 +70,7 @@
     "babel-core": "6.26.0",
     "babel-preset-env": "1.6.1",
     "body-parser": "1.18.3",
+    "coffeescript": "1.12.7",
```

Comment:
Once we have it in the tool chain only to verify `--require=coffeescript/register` works, we could actually upgrade it to 2.x to get rid of the annoying warnings. But that's just a note for future from the top of my head, nothing to care about now.

Reply:
Yeah I thought the same but I think this is better left for a separate PR.